include/translate-option.php

```diff
@@ -130,13 +130,16 @@ public function translate( $value ) {
 	 *
 	 * @param mixed      $values Either a string to translate or a list of strings to translate.
```

Comment:
This should avoid to add `@var` inside the method.
```suggestion
	 * @param string|array|Traversable $values Either a string to translate or a list of strings to translate.
```

Reply:
If we change things, that will be in a separate PR.